src/components/Spawn/index.ts

```diff
@@ -0,0 +1,3 @@
+import Spawn from './Spawn';
```

Comment:
I think this file isn't necessary since we can import the component from `Spawn.tsx` file.

Reply:
We decided to have index.ts for components as I remember, that's I have it like that.